src/Tizen.NUI/src/internal/Interop/Interop.ControlDevel.cs

```diff
@@ -104,28 +104,12 @@ internal static partial class ControlDevel
             public static extern void DaliToolkitDevelControlClearAccessibilityAttributes(global::System.Runtime.InteropServices.HandleRef arg1);
 
             [EditorBrowsable(EditorBrowsableState.Never)]
-            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_SetAccessibilityReadingInfoType")]
-            public static extern void DaliToolkitDevelControlSetAccessibilityReadingInfoType(global::System.Runtime.InteropServices.HandleRef arg1, Tizen.NUI.BaseComponents.ReadingInfoTypes arg2);
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Toolkit_DevelControl_SetAccessibilityReadingInfoType2")]
```

Comment:
could you update the csharp-binder's patch on this PR? and csharp-binder patch should be merged prior to this PR. otherwise there can be a runtime exception which blocks the release of tizenfx by TCT hurdle

Reply:
This function exists in dali-csharp-binder (on devel/master branch). No patch should be required.